### DISCUSSION TITLE
Compress directory backups as a single file

### DISCUSSION THREAD

**Discussion by midestefanis** [prodrigestivill/docker-postgres-backup-local#138] [Q&A]
I leave an example of a small DB:

![image](https://user-images.githubusercontent.com/59846928/92412091-ef0bd900-f120-11ea-96da-9ac132437c66.png)

The last backup is with the new configuration and, as can be seen, it increases the size by 166%.

![image](https://user-images.githubusercontent.com/59846928/92412132-182c6980-f121-11ea-9bb4-2db0f2694108.png)

Then if I compress that directory. The size happens to be similar to the previous ones (which is cheaper for me to store in the cloud, especially when we talk about GBs of storage and multiple DBs).

```
tar -zcf ****-20200904-173736-dump.tar.gz ***-20200904-173736-dump
2.4M    ***-20200904-041500.sql.gz
6.4M    ***-20200904-173736-dump
2.6M    ***-20200904-173736-dump.tar.gz
```

Besides being cheaper in my opinion, it is also easier to manipulate a compressed file than a folder, don't you consider the same?

It is only a suggestion and an exchange of opinion. I can solve it by compressing that directory myself, but I was interested in sharing my point of view with you since this project is looking very good. Let me know if you think the same as me!

Originally posted by @midestefanis in #28 (comment)

**Comment by prodrigestivill** [comment]
Isn't using `POSTGRES_EXTRA_OPTS: -Ft -Z6` already fixes this issue?
